interceptor/reflection.go

```diff
@@ -24,6 +25,10 @@ var (
 		"EventsBatches":  true, // HistoryTaskAttributes
 		"HistoryBatches": true, // GetWorkflowExecutionRawHistoryV2
 	}
+
+	searchAttributeFieldNames = map[string]bool{
+		"SearchAttributes": true, // UpsertWorkflowSearchAttributesEventAttributes, WorkflowExecutionStartedEventAttributes
```

Comment:
also ExecutionInfo in  SyncWorkflowStateTaskAttributes

Reply:
already handled. I have unit test with the ExecutionInfo field: https://github.com/temporalio/s2s-proxy/blob/8a4f7685423f91c6d6373e6333212487cba61343/interceptor/search_attribute_translator_test.go/#L59-L68